src/screens/shared/farmProfileEdit/FarmProfileEditForm.js

```diff
@@ -34,59 +56,65 @@ class FarmProfileEditForm extends React.PureComponent {
 
   render() {
     const { values, classes } = this.props;
+    const { farm, farmId } = this.state;
+    console.log(farmId);
+
     return (
       <div className={classes.root}>
         <h2>Contact Information</h2>
         <div className={classes.form}>
           <div className={classes.row}>
             <FieldInput
               label="First Name"
-              onChange={this.onChange('firstName')}
-              placeholder="Andi"
+              onChange={this.onChange('contactFirstName')}
+              placeholder={farm.contactFirstName}
               variant="outlined"
             />
             <FieldInput
               label="Last Name"
-              onChange={this.onChange('lastName')}
-              placeholder="Halim"
+              onChange={this.onChange('contactLastName')}
+              placeholder={farm.contactLastName}
               variant="outlined"
             />
           </div>
           <div className={classes.row}>
             <FieldInput
               label="Farm Name"
               onChange={this.onChange('farmName')}
-              placeholder="Andi's Potato Farm"
+              placeholder={farm.farmName}
```

Comment:
can you change the placeholder prop here to be `value={farm.farmName}` etc for all farm props.

And replace the `placeholder={placeholder}` line 31 in `FieldInput`.

Reply:
resolved.